src/tabs/tab-header.component.ts

```diff
@@ -48,13 +49,23 @@ export class TabHeader {
 	 * Reference to the corresponsing tab pane.
 	 */
 	@Input() paneReference: Tab;
+	/**
+	 * Set to 'true' to have pane reference cached and not reloaded on tab switching.
+	 */
+	@Input() cacheActive = false;
 	/**
 	 * Value 'selected' to be emitted after a new `Tab` is selected.
 	 */
 	@Output() selected = new EventEmitter<any>();
 
 	@ViewChild("tabItem") tabItem: ElementRef;
 
+	ngAfterViewInit() {
+		if (this.paneReference) {
+			this.paneReference.cacheActive = this.cacheActive;
```

Comment:
This should be done in a way that it updates with changes in cacheActive during the lifetime of the component.